openage/testing/testlist.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2015-2017 the openage authors. See copying.md for legal info.
+# Copyright 2017-2017 the openage authors. See copying.md for legal info.
```

Comment:
Good question. It should remain 2015. We disabled the start-date-check a while ago because of performance reasons parsing the whole git history for every single file to figure out its oldest date.